src/main/java/mate/academy/spring/service/UserService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public interface UserService {
+    void add(User user);
```

Comment:
why void? Lets return User

Reply:
Void is in condition of this task. It's not my fault)))
  public interface UserService {
      void add(User user);
  
      List<User> getAll();
  }

Reply:
![image](https://user-images.githubusercontent.com/88316374/148422702-43891fc6-5a1e-471b-8a9d-db1cafe00987.png)